templates/_footer.phtml

```diff
@@ -3,7 +3,11 @@
                 <div class="col-md-12">
                     <hr/>
                     <p>
-                        Copyright &copy; <?php echo esc(date("Y")); ?> The Godot Engine community - MIT licensed
+                        Copyright &copy; <?php echo esc(date("Y")); ?> The Godot Engine community - MIT licensed 
+                        <?if(isset($user)):?>
+                            <br><a href="https://godotengine.org/asset-library/change_password">Change Password</a>
+                        <?php endif; ?>
```

Comment:
Use `if (...) {` and `}` instead of `if (...):` and `endif` (since it is done this way in other places in the project).